mvcc/kvstore.go

```diff
@@ -160,6 +166,56 @@ func (s *store) Hash() (hash uint32, revision int64, err error) {
 	return h, s.currentRev, err
 }
 
+func (s *store) HashByRev(rev int64) (hash uint32, currentRev int64, compactRev int64, err error) {
+	readTx := s.b.ReadTx()
+	for {
```

Comment:
I wonder is waiting for compaction even necessary?
The moment i obtain the  `txnRead` I have the view of the world; i know the compaction rev, current rev.

Then could I just hash from all the ways between [compaction rev, current rev]?